models/v3/msgs/msgs.go

```diff
@@ -141,7 +146,20 @@ func (n Notifications) dataToJSON() ([]byte, error) {
 
 // SendEvent converts the notification to an event and sends it to the ARN service.
 // Do not call this function directly, use methods on the Client instead.
-func (n Notifications) SendEvent(hc *http.Client, store *storage.Client) error {
+func (n Notifications) SendEvent(hc *http.Client, store *storage.Client) (reterr error) {
```

Comment:
reterr doesn't really do it.  

This should be (err error).

Convert:
if err := event.Validate(); err != nil {
		return err
	}

to:

if err = event.Validate(); err != nil {
		return err
	}

Everything else should work fine (assuming you change the defer to use err now).

Reply:
thank you for catching